crates/kitsune_p2p/transport_quic/src/lib.rs

```diff
@@ -6,46 +6,3 @@ pub mod dependencies {
     pub use ::kitsune_p2p_types;
```

Comment:
@maackle - the whole transport_quic crate should be deleted, it is only used by tx2.